index.js

```diff
@@ -65,18 +65,45 @@ menuLinkContactElement.addEventListener('click', () => {
 
 // ----------------------- contact validation ---------------------------------
 const formElement = document.getElementById('contact-form');
-// const formNameElement = document.getElementById('name');
+const formNameElement = document.getElementById('name');
 const formEmailElement = document.getElementById('email');
 
 formElement.addEventListener('submit', (event) => {
   const email = formEmailElement.value;
   const errmessages = [];
   if (email.toLowerCase() !== email) {
     event.preventDefault();
-    errmessages.push('email must all be in lowercase letters. form was not submitted');
+    errmessages.push(
+      'email must all be in lowercase letters. form was not submitted',
+    );
     const emailErrElement = document.getElementById('email-err');
-    // Destructure the entire `errmessages` array and access the first element using an index
     const [firstErrorMessage] = errmessages;
     emailErrElement.innerText = firstErrorMessage;
   }
 });
+
+const formData = {
+  commentElement: document.getElementById('comment-box'),
+};
```

Comment:
- [ ] The form data for all inputs should be stored in a single object, and saved to a single local storage registry, kindly make sure all the data are stored in only one object, as for now every input is stored in a separate local storage

![Screenshot_20230106_212258](https://user-images.githubusercontent.com/25227022/211093130-5d6373c8-5acc-440f-b00d-f7ce0695aeec.png)